src/routes/(app)/+layout.svelte

```diff
@@ -113,14 +113,41 @@
 
 	$: logoClass = data.logo.isWide ? 'h-[60px] w-auto' : 'h-[60px] w-[60px] rounded-full';
 	const { t, locale, textAddress } = useI18n();
+	let ageWarning: false;
 </script>
 
 <!--
 	We use data-sveltekit-preload-data="off" on header/footer links due to this: 	https://github.com/sveltejs/kit/issues/9508#issuecomment-1807200239
 -->
 
 <div data-sveltekit-preload-data={data.isMaintenance ? 'tap' : 'hover'} style="display: contents;">
-	{#if $page.data.layoutReset || $page.url.searchParams.get('display') === 'headless'}
+	{#if data.ageRestriction && !data.sessionAcceptAgeLimitation}
+		<form class="mx-auto max-w-7xl mt-8" method="POST" action="?/navigate">
+			<h1 class="text-xl">Attention</h1>
+			<p class="my-4">
+				Ce site vend du cul, des clopes, de l'alcool, et des streamings de Ghostbusters avec des
+				protagonistes qui fument, boivent, se font faire des fellations par des fantômes, et
+				prononcent des injures.
+			</p>
+			<p class="my-4">
+				En validant ce formulaire pour continuer la navigation, vous admettez avoir l'âge légal dans
+				votre pays.
+			</p>
+			<p class="my-4">
+				On fait ça parce que <a href={data.ageLegalReason} class="underline">telle page</a> nous le demande
+			</p>
+			<label class="label-checkbox my-4">
+				<input type="checkbox" class="form-checkbox" bind:checked={ageWarning} />
+				Compris chef
+			</label>
```

Comment:
Uhhh I don't think we want this as IS!!

This was just an example.

It should be configurable (maybe CMS page)